src/components/randomize.jsx

```diff
@@ -44,8 +44,15 @@ export default class Randomize extends React.Component {
   }
 
   render() {
-    return (
-      <button className="button" className="btn btn-outline-danger btn-sm" onClick={this.handleClick}>Randomize Colors</button>      
-    )
+    if(this.props.enabled == "enabled"){
+      return (
+        <button className="button" className="btn btn-outline-danger btn-sm" onClick={this.handleClick}>Randomize Colors</button>
```

Comment:
So this is nitpicky and was an issue in the previous code, but className is repeated twice?